openstack_controller/datadog_checks/openstack_controller/api.py

```diff
@@ -210,6 +210,13 @@ def get_project_limits(self, project_id):
         for raw_value, value in key_name_conversion.items():
             project_limits[value] = project_limits_raw[raw_value]
 
+        try:
+            network_quotas = self.connection.get_network_quotas(project_id, details=True)
+            project_limits['totalFloatingIpsUsed'] = network_quotas['floatingip']['used']
+            project_limits['maxTotalFloatingIps'] = network_quotas['floatingip']['limit']
+        except Exception:
```

Comment:
AFAIU the only  `Exception` we expect here would be a `KeyError`, maybe we should be more specific

Reply:
I'm actually most worried about the `get_network_quotas` call to fail on older versions. I wasn't able to test it though.

Reply:
Maybe sth like
```
        except KeyError:
            pass
        except Exception:
            self.log.exception("There was a problem getting network quotas")
```

Reply:
Tried something.